packages/mobile/src/in-house-liquidity/index.ts

```diff
@@ -40,13 +40,18 @@ const getSerializedSignature = async (
   const signature = signMessage(message, privateKey, accountMTWAddress)
   return serializeSignature(signature)
 }
-const getAuthHeader = async (
+
+const getAuthAndDateHeaders = async (
   endpoint: string,
   accountMTWAddress: string,
   body: string = ''
-): Promise<string> => {
-  const dateHeader = new Date().getUTCDate()
-  const message = `${endpoint} ${dateHeader} ${body}`
+): Promise<{ Date: string; Authorization: string }> => {
+  const date = new Date().toUTCString()
+  const message = `${endpoint} ${date} ${body}`
```

Comment:
this actually won't work for the GET case because it will have a trailing space that isn't expected

Reply:
let me refactor it a bit and actually write some unit tests for it too